src/main/kotlin/com/craftschemers/hub/managers/HubMinigameManager.kt

```diff
@@ -13,15 +13,16 @@ class HubMinigameManager {
     fun addPlayerToMinigame(player: HubPlayer, gameType: GameType) {
         // make sure a server is available
         val gameServer = minigameServers[gameType]
-        if (gameServer == null || gameServer.none { it.state == GameState.LOBBY }) {
+        val nonStartedGame = gameServer?.find { it.state == GameState.LOBBY }
+        if (gameServer == null || nonStartedGame == null) {
```

Comment:
since `nonStartedGame` is equal to `gameServer?.find ...`, if `gameServer` is null so is `nonStartedGame` so this double null check is redundant.
```suggestion
        if (nonStartedGame == null) {
```